<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetApplicationSettings">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRecordsettings">
      <attribute name="title">
       <string>Record settings (ffmpeg)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelRecordheader">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Record settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>451</width>
            <height>402</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Basic</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QLabel" name="labelfps">
               <property name="text">
                <string>Frames pr. sec</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spinBoxfps">
               <property name="toolTip">
                <string>Changes the framerate in the recording</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labelvideocodec">
               <property name="text">
                <string>Video codec</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEditvideocodec">
               <property name="toolTip">
                <string>Changes the video codec used in the recording</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="labelaudiocodec">
               <property name="text">
                <string>Audio codec</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEditaudiocodec">
               <property name="toolTip">
                <string>Changes the audio codec used in the recording</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="labelaudiochannles">
               <property name="text">
                <string>Audiochannels</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QSpinBox" name="spinBoxaudiochannels">
               <property name="toolTip">
                <string>Changes the number of audiochannels used. To record in mono set to 1, for stereo set to 2 etc.</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="labelRecordingdevice">
               <property name="text">
                <string>Microphone</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="2">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="labelbasename">
               <property name="text">
                <string>Basename</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLineEdit" name="lineEditbasename">
                 <property name="toolTip">
                  <string>Choose a basename for your recordings.</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="checkBoxbasenametimedate">
                 <property name="toolTip">
                  <string>Use the time and date as the basename.</string>
                 </property>
                 <property name="text">
                  <string>Time/Date</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="9" column="1">
              <widget class="QLabel" name="labelpath">
               <property name="text">
                <string>Default path</string>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLineEdit" name="lineEditpath">
                 <property name="toolTip">
                  <string>Choose where to save your recordings</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButtonpathBrowse">
                 <property name="toolTip">
                  <string>Change the default path.</string>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="10" column="1">
              <widget class="QLabel" name="labelFormat">
               <property name="text">
                <string>Default format</string>
               </property>
              </widget>
             </item>
             <item row="10" column="2">
              <widget class="QComboBox" name="comboBoxFormat">
               <property name="toolTip">
                <string>Choose which format the recordings should be saved with. </string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QComboBox" name="comboBoxrecording">
                 <property name="toolTip">
                  <string>Choose the microphone you want to use.</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEditAudiosource">
               <property name="toolTip">
                <string>Changes the sound architecture used in the recording. Thjs can be alsa, oss etc.</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Sound architecture</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QCheckBox" name="checkBoxMicMute">
               <property name="toolTip">
                <string>The microphone is muted as default on startup</string>
               </property>
               <property name="text">
                <string>Mute on startup</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="Pixmaplabel_microphone">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/devices/audio-input-microphone.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="Pixmaplabel_Defaultpath">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/places/folder-video.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="Pixmaplabel_videocodec">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/devices/video-display.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="Pixmaplabel_videocodec_2">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/apps/preferences-desktop-sound.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="Pixmaplabel_Defaultformat">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/mimetypes/video-x-generic.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="Pixmaplabel_basename">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../../../../usr/share/icons/default.kde4/22x22/actions/edit-rename.png</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>44</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>451</width>
            <height>402</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Advanced</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="labeladvancedapre">
               <property name="text">
                <string>-apre</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditadvancedapre"/>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="checkBoxadvancedapre">
               <property name="text">
                <string>Use</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="labeladvancedvpre">
               <property name="text">
                <string>-vpre</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditadvancedvpre"/>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="checkBoxadvancedvpreuse">
               <property name="text">
                <string>Use</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>314</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabApplicationSettings">
      <attribute name="title">
       <string>Application Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelAppSettingsheader">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Application settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="1">
          <widget class="QLabel" name="labelPreviewPlayer">
           <property name="text">
            <string>Preview player</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditPreviewplayer"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelLanguage">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBoxLanguage"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Pixlabel_previewplayer">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../usr/share/icons/oxygen/22x22/actions/player-time.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Pixmaplabel_Language">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../../../../usr/share/icons/oxygen/22x22/categories/applications-education-language.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBoxPreviewplayer">
           <property name="text">
            <string>Use integrated player</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>278</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="checkBoxSinglewindowredrectangle">
           <property name="text">
            <string>Enable/Disable</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="labelSinglewindowredrectangle">
           <property name="text">
            <string>Show red rectangle</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="labelSinglewindowHeader">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Single Window</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>189</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>NOTE: Some changes might first apply after a restart of this program</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="statusTip">
        <string>Save or decline changes</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonRestore">
       <property name="toolTip">
        <string>Restores the settings to the defaults. Please note, these can not be undone!</string>
       </property>
       <property name="statusTip">
        <string>Restore to default. PLEASE NOTE: CAN NOT BE UNDONE!</string>
       </property>
       <property name="text">
        <string>Restore to default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
